package.json

```diff
@@ -3,7 +3,7 @@
   "author": "Dassault Systèmes SE",
   "description": "The official NuoDB database driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "BSD-3-Clause",
-  "version": "3.7.1",
+  "version": "3.7.1.1",
```

Comment:
I will not publish to NPM until I am ready to bump to 3.7.2, I just need this drop for 3DMessage to test since we do not have a reproducible test case.